<launch>
	<node pkg="mavlink_sysu" type="onboard_station2" name="onboard_station2" output="screen">
		<param name="use_udp" value="true" type="bool"/>
		<param name="udp_ip" value="127.0.0.1" type="string"/>
		<param name="rx_port" value="14541" type="int"/>
		<param name="tx_port" value="14540" type="int"/>
	</node>

	<node pkg="mavlink_sysu" type="ground_station2" name="ground_station2" output="screen" launch-prefix="gnome-terminal --">
		<param name="use_udp" value="true" type="bool"/>
		<param name="udp_ip" value="127.0.0.1" type="string"/>
		<param name="rx_port" value="14540" type="int"/>
		<param name="tx_port" value="14541" type="int"/>
	</node>


</launch>

